src/modules/solution-explorer/solution-explorer-list/solution-explorer-list.ts

```diff
@@ -101,7 +87,7 @@ export class SolutionExplorerList {
   }
 
   public async openSingleDiagram(uri: string): Promise<IDiagram> {
-    const identity: IIdentity = this._createIdentityForSolutionExplorer();
+    const identity: IIdentity = await this._createIdentityForSolutionExplorer();
```

Comment:
```suggestion
    const identity: IIdentity = this._createIdentityForSolutionExplorer();
```

Remove this await if the comment below on this called method is right